.github/workflows/sanitize.yml

```
@@ -14,96 +14,182 @@ concurrency:
+          path: /FalkorDB/bin/linux-x64-debug-asan/libcypher-parser
+          key: parser-sanitizer-llvm-${{ hashFiles('./deps/libcypher-parser/lib/src/parser.c') }}
+
+      - name: Cache search
+        id: cache_search
+        uses: actions/cache@v4
+        with:
+          path: /FalkorDB/bin/linux-x64-debug-asan/search-static
+          key: search-sanitizer-llvm-${{ hashFiles('./deps/RediSearch/src/version.h') }}
+
+      - name: Relocate source
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:9: Double quote to prevent globbing and word splitting

(shellcheck)